manifests/tasks/retrieve-build-image-info/retrieve-build-image-info.yaml

```diff
@@ -26,9 +29,6 @@ spec:
   - name: target-image-tag-references
     type: array
     description: The image tag references used for the final built image
-  workspaces:
-  - name: images-url
-    description: workspace where url.txt file is created
```

Comment:
I think in this task we just need to change the workspace name to `images_url` and that's all.